test/integration/index.js

```diff
@@ -4,18 +4,23 @@ const knex = require('../../knex');
 const logger = require('./logger');
 const config = require('../knexfile');
 const fs = require('fs');
+const path = require('path');
 
 const Bluebird = require('bluebird');
 
 Object.keys(config).forEach((dialectName) => {
   return require('./suite')(logger(knex(config[dialectName])));
 });
 
-after(function(done) {
+before(function(done) {
   if (config.sqlite3 && config.sqlite3.connection.filename !== ':memory:') {
-    fs.unlink(config.sqlite3.connection.filename, function() {
-      done();
-    });
+    fs.copyFile(
```

Comment:
Synchronous version would be preferable here, we don't really care about performance in this case.